src/core/packages/saved-objects/api-server-internal/src/lib/apis/utils/internal_utils.ts

```diff
@@ -38,7 +38,7 @@ export function getBulkOperationError(
   id: string,
   rawResponse: {
     status: number;
-    error?: { type: string; reason?: string; index: string };
+    error?: { type: string; reason?: string | null; index: string };
```

Comment:
Sorry, this is annoying, but we have been seeing null reasons in search and ES|QL recordings: https://github.com/elastic/clients-flight-recordings/blob/3cb95e68f2ef95ccfca8f08b3a284bd31322e6f6/elasticsearch/esql.query/878a05ca4333c3b1772752a5cb06109e.json#L22